lib/node_modules/@stdlib/math/generics/random/sample/lib/factory.js

```diff
@@ -231,6 +259,7 @@ function factory() {
 				throw err;
 			}
 		}
+		mutate = opts.mutate !== void 0 ? opts.mutate : conf.mutate;
```

Comment:
Based on the style guide and instead of requiring a negation when performing an equality check,

```
mutate = ( opts.mutate === void 0 ) ? conf.mutate : opts.mutate;
replace = ( opts.replace === void 0 ) ? conf.replace : opts.replace;
```